azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -392,6 +394,12 @@ private boolean canWriteInCurrentWorkingDirectory(final String effectiveUser)
     final List<String> checkIfUserCanWriteCommand = Arrays
         .asList(CREATE_FILE, getWorkingDirectory() + "/" + TEMP_FILE_NAME);
     final int result = executeAsUser.execute(effectiveUser, checkIfUserCanWriteCommand);
+    // If TEMP_FILE user_can_write is created, it should be deleted at the end of the function
+    try {
+      Files.deleteIfExists(Paths.get(getWorkingDirectory() + "/" + TEMP_FILE_NAME));
+    } catch (Exception e) {
+      info(String.format("Failed to delete %s in current working directory", TEMP_FILE_NAME));
```

Comment:
Include `e` in the log? So that we can tell why it failed.

Reply:
Included e in the log.